.github/workflows/ci-linux-shared-local-fuzztest.yml

```diff
@@ -0,0 +1,68 @@
+# Builds libavif with local aom and dav1d and runs tests including fuzztest.
+# Runs on ubuntu only. libavif is built with clang.
+
+name: CI Unix Shared Local Fuzztest
+on: [push, pull_request]
+
+permissions:
+  contents: read
+
+# Cancel the workflow if a new one is triggered from the same PR, branch, or tag, except on main.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+
+jobs:
+  build-shared-local:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-24.04]
+        libyuv: [OFF]
+        include:
+          - runs-on: ubuntu-24.04
+            compiler: gcc
+            gcc: 14
+
+    name: build-shared-local-fuzztest
+
+    steps:
+    - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+    - uses: ./.github/actions/setup-linux
+      id: setup
+      with:
+        codec-aom: 'LOCAL'
+        codec-dav1d: 'LOCAL'
+        gcc-version: ${{ matrix.gcc }}
+        libxml2: 'LOCAL'
+        libyuv: 'LOCAL'
+        recent-cmake: 'true'
+    - name: Build fuzztest
+      if: steps.setup.outputs.ext-cache-hit != 'true'
+      working-directory: ./ext
+      run: bash -e fuzztest.cmd
+
+    - name: Prepare libavif (cmake)
+      run: >
+        cmake -G Ninja -S . -B build
+        -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON
+        -DAVIF_CODEC_AOM=LOCAL -DAVIF_CODEC_DAV1D=LOCAL
+        -DAVIF_LIBSHARPYUV=LOCAL -DAVIF_LIBXML2=LOCAL
+        -DAVIF_LIBYUV=${{ matrix.libyuv }}
+        -DAVIF_BUILD_EXAMPLES=ON -DAVIF_BUILD_APPS=ON
+        -DAVIF_BUILD_TESTS=ON -DAVIF_ENABLE_GTEST=ON -DAVIF_GTEST=LOCAL
+        -DAVIF_ENABLE_EXPERIMENTAL_YCGCO_R=ON
+        -DAVIF_ENABLE_EXPERIMENTAL_GAIN_MAP=ON
+        -DAVIF_ENABLE_EXPERIMENTAL_MINI=ON
+        -DAVIF_ENABLE_EXPERIMENTAL_SAMPLE_TRANSFORM=ON
+        -DAVIF_ENABLE_FUZZTEST=ON
+        -DAVIF_LOCAL_FUZZTEST=ON
+        -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++
+        -DAVIF_ENABLE_WERROR=ON
+    - name: Build libavif (ninja)
+      working-directory: ./build
+      run: ninja
+    - name: Run AVIF Tests
+      working-directory: ./build
+      run: ctest -j $(getconf _NPROCESSORS_ONLN) --output-on-failure
```

Comment:
Should this be limited to fuzz tests only?

Reply:
Done